pkg/services/annotations/annotations.go

```diff
@@ -15,6 +15,7 @@ var (
 
 type Repository interface {
 	Save(ctx context.Context, item *Item) error
+	SaveMany(ctx context.Context, items []Item) error
```

Comment:
I felt this was a cleaner option than making `Save` variadic. See the comment in `xorm_store.go` as to details on the DB-level limitations here. Two separate methods wasn't my initial approach.

Background: Some callers of `Save()` currently rely on XORM auto-populating the `item.Id` field through the pointer. For reasons below, we can't do this when batch inserting.

Since `SaveMany` not fill the ID field on the items, accepting a `[]Item` over an `[]*Item` forces the caller to recognize this. IMO having a variadic `Save` silently/conditionally not fill IDs if multiple items are provided is just _asking_ for bugs. In this approach, each method has one singular behavior, which is less accident prone.